sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusMessageSerializerTest.java

```diff
@@ -132,6 +139,69 @@ void deserializeMessage() {
         assertEquals(payload, actualMessage.getBody().toString());
     }
 
+    /**
```

Comment:
Is there a repro from the original message? One way we did this was sending a message via the .NET application, and when we received the message on this side, saving the message as a byte array in this test, so we can double-check that it works correctly.

Reply:
Sure, added.